tabular/src/autogluon/tabular/predictor/predictor.py

```diff
@@ -1950,6 +1950,29 @@ def feature_importance(self, data=None, model=None, features=None, feature_stage
             fi_df[low_str] = pd.Series(ci_low_dict)
         return fi_df
 
+    def compile_models(self, compiler_configs={}):
```

Comment:
Don't use {} as default (mutable), use None and then if compiler_configs is None: compiler_configs = {}

Reply:
Done.